sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsBufferStore.scala

```diff
@@ -201,7 +209,15 @@ abstract class RapidsBufferStore(
    * @param stream CUDA stream to use or null
```

Comment:
The new `memoryBuffer` argument needs to be documented and mention that this method will close it.